ext/data/templates/default-anki-field-templates.handlebars

```diff
@@ -302,6 +302,54 @@
     {{~/if~}}
 {{/inline}}
 
+{{#*inline "frequency-number"}}
+    {{~#scope~}}
+        {{~set "resultFrequency" -1 ~}}
+        {{~set "previousFrequencyDictionary" "" ~}}
+        {{~set "total" 1 ~}}
+
+        {{~#each definition.frequencies~}}
```

Comment:
seems like a lot of logic for handlebars, maybe do this in `anki-note-data-creator.js`?